src/signals/settings/users/containers/Detail/__tests__/Detail.test.js

```diff
@@ -1,19 +1,39 @@
 import React from 'react';
+import { act } from 'react-dom/test-utils';
 import { render } from '@testing-library/react';
 import { withAppContext } from 'test/utils';
 import routes from 'signals/settings/routes';
 
 import UserDetail from '..';
 
 describe('signals/settings/users/containers/Detail', () => {
-  it('should render a backlink', () => {
+  it('should render a backlink', async () => {
     const referrer = '/some-page-we-came-from';
-    const { container, rerender } =  render(withAppContext(<UserDetail location={{}} />));
+    let container;
+    let rerender;
+
+    await act(async () => {
+      ({ container, rerender } = await render(withAppContext(<UserDetail location={{}} />)));
+    });
 
     expect(container.querySelector('a').getAttribute('href')).toEqual(routes.users);
 
-    rerender(withAppContext(<UserDetail location={{ referrer }} />));
+    await act(async () => {
+      rerender(withAppContext(<UserDetail location={{ referrer }} />));
+    });
 
     expect(container.querySelector('a').getAttribute('href')).toEqual(referrer);
   });
+
+  it('should show an alert on error', async () => {
+    let getByText;
+    const error = new Error('Not allowed');
```

Comment:
Could we also test this message `toBeInTheDocument()`?

Reply:
The message itself won't be in the document